fish_prompt.fish

```diff
@@ -20,7 +20,7 @@ function fish_prompt
   set -l none     "◦"
 
   set -l normal_color     (set_color normal)
-  set -l success_color    (set_color $fish_pager_color_progress 2> /dev/null; or set_color cyan)
+  set -l success_color    (set_color cyan)
```

Comment:
I couldn't avoid asking, how about using `$fish_color_param`?

```fish
 ~> echo $fish_color_param
cyan
```

Reply:
Hmm, not bad. Seems like a fair plan to me!

Reply:
I find pretty weird that fish defines an error color variable but no success error one.